content/stopping-systemd-in-openshift.md

```diff
@@ -280,9 +277,9 @@ spec:
 ```
 
 - (1) The lifecycle hooks for that container.
-- (2) Indicate that will be called before stopping the container.
-- (3) It will be an exec command.
-- (4) The command to be executed; the command should exist inside the container.
+- (2) A `preStop` hook is called before stopping the container.
+- (3) It will be an `exec` command.
+- (4) The command to be executed; the executable must exist in the container.
 
```

Comment:
Note that the rendered HTML for this list is a bit strange (extra space beween some of the bullets).  But I decided not to try and fix that.  Just giving you the heads-up.

Reply:
Thanks @frasertweedale ; Pelican is not rendering this as I was expecting; I have tried with cobalt and that is working better when rendering the resulting html. I will move to [cobalt](https://cobalt-org.github.io) as soon as possible.